src/frontend/src/hooks/auth.hooks.ts

```diff
@@ -16,10 +16,14 @@ export const useProvideAuth = () => {
   const [user, setUser] = useState<AuthenticatedUser | undefined>(undefined);
 
   const devSignin = async (userId: number) => {
-    const user = await mutateAsyncDev(userId);
-    setUser(user);
-    localStorage.setItem('devUserId', userId.toString());
-    return user;
+    try {
```

Comment:
just remove the try, catch, since it's not doing anything

Reply:
Ah, thanks! I had to revert some changes locally and lost my work and must've missed fixing that when I went back to work on it aughh